sql-cli/sql_cli/__main__.py

```diff
@@ -38,5 +41,53 @@ def generate(
     rprint("The DAG file", dag_file.resolve(), "has been successfully generated. 🎉")
 
 
+@app.command()
+def init(project_dir: Optional[str] = typer.Argument(None)) -> None:
+    """
+    Initialise a SQL CLI project structure.
+
+    By default, this includes:
+
+    \b\n
```

Comment:
I think using `\b` instead of `\b\n` should also work.

Reply:
It didn't work - that was my first attempt. ATM the code looks disgusting - but the help looks okay:
```
$ poetry run flow init --help
                                                                                                         
 Usage: flow init [OPTIONS] [PROJECT_DIR]                                                                
                                                                                                         
 Initialise a SQL CLI project structure.                                                                 
 By default, this includes:                                                                              
                                                                                                         
                                                                                                         
 ├── config                                                                                              
 │   ├── default                                                                                         
 │   │   └── configuration.yml                                                                           
 │   └── dev                                                                                             
 │       └── configuration.yml                                                                           
 ├── data                                                                                                
 │   ├── movies.db                                                                                       
 │   └── retail.db                                                                                       
 └── workflows                                                                                           
 ├── example_basic_transform                                                                             
 │   └── top_animations.sql                                                                              
 └── example_templating                                                                                  
 ├── filtered_orders.sql                                                                                 
 └── joint_orders_customers.sql                                                                          
                                                                                                         
                                                                                                         
 Update the file `config/default/configuration.yaml` to declare your databases.                          
 Create SQL workflows within the `workflows` folder.                                                     
                                                                                                         
╭─ Arguments ───────────────────────────────────────────────────────────────────────────────────────────╮
│   project_dir      [PROJECT_DIR]  [default: None]                                                     │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────╯
╭─ Options ─────────────────────────────────────────────────────────────────────────────────────────────╮
│ --help          Show this message and exit.                                                           │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────╯
```
I'll follow your advice to see if we can get the directory tree using rich :)

Reply:
https://rich.readthedocs.io/en/stable/tree.html

Reply:
@feluelle for now I simplified the docstring - we can improve it in future PRs.